src/test/java/racing/RacingTest.java

```diff
@@ -9,31 +9,43 @@
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static racing.InputView.*;
+import static racing.Validate.*;
 
 public class RacingTest {
     @DisplayName("자동차 대수와 이동 횟수는 자연수이고 2대 이상이어야 함")
     @ParameterizedTest
     @ValueSource(ints = {2, 3, 4, 5})
-    void 입력값_확인(int input){
-        validateInput(input);
-    }
+    void 입력값_확인(int input){ validInputMoveCount(input);}
 
     @DisplayName("2개의 차가 달리고 2번째 차량이 움직인 경우 테스트")
     @Test
     void 정상_작동_확인(){
-        Strategy strategy = new RandomStrategy();
-        List<Car> carList = new ArrayList<>();
-        for(int index = 0; index < 2; ++index){
-            carList.add(new Car());
+        Strategy forwardStrategy = new Strategy(){
+            @Override
+            public boolean isMovable(){
+                return true;
+            }
+        };
```

Comment:
```suggestion
        Strategy forwardStrategy = () -> true;
```

인터페이스에 있는 메서드의 반환 타입과 파라미터를 봤을 때
`boolean isMovable();` 입력값은 없고 반환 타입은 boolean이라는 것을 확인할 수 있습니다.
이렇게 인터페이스에서 구현해야 할 메서드가 하나이고 타입, 파라미터가 뭐가 들어가야 하는지 알 수 있기 때문에 추론이 가능합니다.
추론이 가능한 부분은 위 람다식을 사용하여 익명 클래스를 대체할 수 있습니다.